src/components/Link.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Link = ({ place, onClick }) => (
+  <a
+    className="dropdown__list__item"
+    onClick={onClick}
+    onKeyDown={onClick}
+    tabIndex={0}
+    role="button"
+  >
+    {place}
+  </a>
+);
+
+Link.propTypes = {
+  place: PropTypes.string.isRequired,
+  onClick: PropTypes.func
```

Comment:
what if the consumer of your Link doesn't pass onClick function? The code will fail. Either use .isRequried for you prop types or default props. That is applicable everywhere else.